### DISCUSSION TITLE
Circle size is terrible on android

### DISCUSSION THREAD

**Discussion by KrispyKebaeb** [ppy/osu#17009] [Ideas] [answered]
It probably is fine on tablets but for phones, the circle size is terrible, is there a reason for this being a thing? Will it be fixed? We can change the circle size with difficulty adjust but once the game is ranked i think that will result in less scores, if the circle change happens it will be way easier to play on mobile phones as they are more common than tablets.

**Comment by bdach** [comment]
it's known. it will be addressed when the mobile version becomes a bigger priority. for now we are focused mostly on the mobile version.

**Reply by Walavouchey** [reply]
desktop version* for the latter

**Comment by peppy** [comment, marked as answer]
For now you can turn on "Difficulty Adjust" mod and adjust it to your will.
